### ISSUE TITLE
consider re-enabling rss feed

### ISSUE THREAD

**Issue by nilesjohnson** [sethjohnson1/conflist5#30]
CakePHP 5.x removed support for RSS.  We could build our own independent support for RSS.  Whether this is worth the effort of designing and maintaining depends on how broadly it's used and whether its features can be replicated in other ways.

Some other relevant features:
- recent announcements now flagged with [NEW]
- json feed still supported
- advanced search by created/modified date

**Comment by nilesjohnson** [comment]
I see there is now UrloverrideHelper instead of RssHelper; seems good.  Here's what seems to be happening:

1. All links for the main lists (current announcements, with or without specific subject tags) omit `/conferences/`, so they are just `<sitename>` or `<sitename>/<tagstring>`

2. The corresponding rss urls shown on the main page likewise just omit the `/conferences/` part

So, all of those rss links (a) match what users would see just clicking links on the main site and (b) have matching `self` references.  This seems like a great solution.  

Manually navigating to some rss feeds like `<site>/conferences/<etc>` gives feeds that are valid, but where the `self` reference doesn't match.  That seems fine by me.  The site never shows those URLS to users, so they shouldn't be considered officially supported even if they work fine.

So, if what I'm seeing is what you intended, then this is ready to merge and close!

**Comment by sethjohnson1** [comment]
Yes, I think we roll with it as-is and leave those cases be. I think we could solve it with another named route.

**NOTE** I think the routes are finally cleaned up, fallbacks disabled once again.
